website/blog/2025-10-15-introducing-atmos-auth-list.md

```
@@ -0,0 +1,187 @@
+NAME      KIND       REGION      START URL                                DEFAULT
+aws-sso   aws-sso    us-east-1   https://example.awsapps.com/start       ✓
+okta      okta                   https://example.okta.com
+
+IDENTITIES
+NAME       KIND              VIA PROVIDER  VIA IDENTITY  DEFAULT  ALIAS
+admin      aws/assume-role   aws-sso                     ✓        prod-admin
+developer  aws/assume-role   aws-sso                              dev
+ops        aws/assume-role   aws-sso       admin                  ops-admin
+```
+
+**Detailed Tree View**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)